Sources/Tuvi.Core.Dec.Bitcoin/Tools.MainNet.cs

```diff
@@ -0,0 +1,66 @@
+﻿using KeyDerivation.Keys;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Tuvi.Core.Dec.Bitcoin.MainNet
+{
+    /// <summary>
+    /// Provides utility methods for Bitcoin-related operations, such as address and key derivation,
+    /// and public key retrieval from the blockchain for MainNet.
+    /// </summary>
+    public static class Tools
+    {
+        static BitcoinNetworkConfig NetworkConfig = BitcoinNetworkConfig.MainNet;
+        static HttpClient HttpClient = new HttpClient();
```

Comment:
It would be nice to be able to create a `HttpClient` object outside `Tuvi.Core.Dec.Bitcoin` module.

Reply:
Done.